ironfish/src/node.ts

```diff
@@ -108,7 +108,8 @@ export class IronfishNode {
       chain: chain,
       strategy: strategy,
       metrics: this.metrics,
-      addressManager: new AddressManager(hosts),
+      files: this.files,
+      dataDir: dataDir,
```

Comment:
Can you look into this, something doesn't seem right here. It doesn't seem like it's ever being passed into the constructor.

Reply:
You're correct, it was not; fixed.